src/graphics/program-lib/programs/standard.js

```diff
@@ -655,6 +655,13 @@ const standard = {
             chunks = customChunks;
         }
 
+        if (options.pass === SHADER_PICK) {
+            code += '#define PICKPASS\n';
```

Comment:
Can we name these with snake case? i.e. PICK_PASS

Reply:
I'd also create a function that converts options.pass to a string .. and just call code += getPassDefine(options.pass);